@hubblecommerce/hubble/core/components/swComponents/slots/product-name-slot.vue

```diff
@@ -18,6 +22,9 @@ export default {
     mixins: [slotMixins],
 
     computed: {
+        isMapped() {
```

Comment:
what means mapped?

Reply:
In Shopware you can declare a data mapping if CMS element is placed on product page.
I will rename this computed variable to 'hasProductDataMapping' to be more precise.

Reply:
Refactored in [91188bf](https://github.com/hubblecommerce/hubble-frontend-pwa/commit/91188bf449e03030a0b9d9b9ac7e9aac82b5f1f8)